M2.el

```diff
@@ -247,10 +247,19 @@ current window added to it."
 
 (defun M2-dynamic-complete-symbol()
   "Dynamic completion function for Macaulay2 symbols."
+  (declare (obsolete completion-at-point "Macaulay2 1.20"))
```

Comment:
Why not remove this function entirely?

Reply:
Just in case someone has `M2-dynamic-complete-symbol` bound to some key combination in their `.emacs`, it would be nice to get a warning message for a release or two before removing it completely.